lib/database/migrations/20240424092020-add-qc-flag-effective-period-columns.js

```diff
@@ -0,0 +1,248 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const { timestampToMysql } = require('../../server/utilities/timestampToMysql');
+
+/**
+ * Discard effective periods of flags belonging to given scope
```

Comment:
This is still not good. You don't discard effective periods of flags belonging to given scope, and you should explicit what is scope

Reply:
refactored